src/net.rs

```diff
@@ -1,54 +1,71 @@
-//! TCP/UDP bindings for `tokio`.
+//! TCP/UDP/Unix bindings for `tokio`.
 //!
-//! This module contains the TCP/UDP networking types, similar to the standard
+//! This module contains the TCP/UDP/Unix networking types, similar to the standard
```

Comment:
I agree that the current documentation should be moved into the appropriate sub module, but that leaves `tokio::net` documentation a bit light, especially given that it is a primary module.

Perhaps, you could add an "orgnaization" section, similar to https://doc.rust-lang.org/std/net/index.html. 

Also, this can mention the unix module and explain that it is only available on unix platforms.

Reply:
Looks like this change was made in 7ea54a01ffee8b06cbfaf887bf15daf9aa696f10.